x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/middleware.test.ts

```diff
@@ -120,6 +123,7 @@ describe('endpoint list middleware', () => {
   });
 
   it('handles `appRequestedEndpointList`', async () => {
+    firstValueFromMock.mockResolvedValue({ indexFields: [] });
```

Comment:
Maybe a new test case for testing what happens with the returned indeces could be useful. What do you think?

Reply:
Well, these indices are set to a component (`search_bar`) that already has tests for it so I would say it's not necessary but I'm open to add it if you feel it is 🙂

Reply:
Do you mean the tests in `search_bar.test.tsx`? The function `firstValue()` is mocked there also, and returns an empty array for `indexFields`, so I'm not sure if it's tested there. Am I missing something?

Reply:
Sorry, I meant the  `search_bar` from the `unified_search` plugin